lib/plugins/aws/package/compile/events/websockets/lib/stage.js

```diff
@@ -39,8 +47,19 @@ module.exports = {
 
       if (!logs) return null;
 
+      const accessLogging = logs.accessLogging == null ? true : logs.accessLogging;
+      const executionLogging = logs.executionLogging == null ? true : logs.executionLogging;
+      const fullExecutionData = logs.fullExecutionData == null ? true : logs.fullExecutionData;
+
+      let logFormat = defaultLogFormat;
```

Comment:
Does it have to be set explicitly if you just want to use default log format? Previously it didn't have to be set explicitly

Reply:
It keeps the previous behavior when there is no options provided by default.
So you don't have to set them explicitly when you just want to use default values.

Reply:
Sorry, I've missed that in fact, we've set these explicitly as well in the past